ush/config.aqm.nco.realtime.yaml

```diff
@@ -34,7 +34,7 @@ nco:
   model_ver: v7.0
   RUN: aqm_nco_aqmna13km
   OPSROOT: /path/to/custom/opsroot
-  KEEPDATA: false
+  KEEPDATA: NO
```

Comment:
Not correct.

Reply:
NCO used YES or NO for KEEPDATA assignment. This is the same with all other models run by NCO.

Reply:
Agree with @lgannoaa here - KEEPDATA is a boolean variable (YES/NO) as described on page 5 of the implementation standards doc.